### PR TITLE
Dark mode: Local navigation

### PR DESCRIPTION
**Pull request by hannahiss**
### Description

Local navigation in dark mode, by using existing css vars:
- `$local-nav-hover-bg` = --bs-secondary-bg // from - `gray300 to gray900 under lg point
- `$local-nav-active-color` = --bs-link-hover-color // from accessible-orange to brand-orange
- `$local-nav-border-color` = --bs-border-color-translucent // from gray-500 to gray-700

⚠️ 3 additional vars are added:
- `--bs-hover-color` to switch `$local-nav-hover-color` from black to brand-orange 
- `--bs-active-bg` to switch `$local-nav-active-bg` from gray-400 to gray-700
- `--bs-disabled-color` to switch `$form-star-rating-disabled-color`, `$nav-link-disabled-color` and `$navbar-light-disabled-color` from gray-500 to gray-700 => ⚠️ **used in local-nav, navbars, navs and star-rating**

### Links

- https://deploy-preview-2276--boosted.netlify.app/docs/5.3/components/local-navigation/
- https://deploy-preview-2276--boosted.netlify.app/docs/5.3/dark-mode/#stepped-process

### CODE REVIEW

File: scss/_root.scss [hunk 1]
```diff
@@ -137,6 +137,12 @@
   --#{$prefix}form-invalid-color: #{$form-invalid-color};
   --#{$prefix}form-invalid-border-color: #{$form-invalid-border-color};
   // scss-docs-end root-form-validation-variables
+
+  // ******* Additions for dark-mode
+  --#{$prefix}hover-color: #{$body-color};
+  --#{$prefix}active-bg: #{$gray-400};
+  --#{$prefix}disabled-color: #{$gray-500};
```
Comment (louismaximepiton) [inline scss/_root.scss:144]: As a first step, I'd be fine fine defining those right there, but IMHO, they should have their own Sass variables somewhere.
Fine for now but if we keep them, I like the naming though it's a bit too evasive maybe.

File: scss/_variables.scss [hunk 2]
```diff
@@ -2356,12 +2356,12 @@ $tag-font-size-sm:                  $font-size-sm !default;
...
-$local-nav-hover-bg:            $gray-300 !default;
-$local-nav-active-color:        $accessible-orange !default;
-$local-nav-active-bg:           $gray-400 !default;
+$local-nav-hover-color:         var(--#{$prefix}hover-color) !default; //from black to brand orange
+$local-nav-hover-bg:            var(--#{$prefix}secondary-bg) !default; // from gray300 to gray900 under lg point
+$local-nav-active-color:        var(--#{$prefix}link-hover-color) !default; // from accessible-orange to brand-orange
+$local-nav-active-bg:           var(--#{$prefix}active-bg) !default; // from gray-400 to gray-700
 $local-nav-active-marker-width: $spacer * .2 !default;
-$local-nav-border-color:        $gray-500 !default;
+$local-nav-border-color:        var(--#{$prefix}border-color-translucent) !default; // from gray-500 to gray-700
```
Comment (louismaximepiton) [inline scss/_variables.scss:2364]: Not quite sure if should keep the comments

**Review by louismaximepiton — APPROVED** [review]
I'm fine as-is, maybe some talks about the variables if we keep them as-is.

🚀

File: scss/_root.scss (hunk 1 above)
Comment (hannahiss) [reply scss/_root.scss:144]: Yes, variables names and way to do it are te be discussed... but for now useful and simple 😄

**Review by julien-deramond — APPROVED** [review]

**Pull request merged**